imports/ui/signup/signup.jsx

```diff
@@ -70,35 +53,38 @@ export class SignupPage extends Component {
     if (!this.isFormValid()) return
 
     const { info } = this.state
-
     // Copying all the input values and trimming them
     const signupInfo = Object.keys(info).reduce((all, curr) => {
       if (info[curr]) {
-        all[curr] = info[curr].trim()
```

Comment:
The usage of "trim" can actually be useful here. I think you should restore it

Reply:
The value of `acceptedAt` is changed from boolean to a string `true`, since trim() throws an error to boolean.

Reply:
You're not sending `acceptedAt` anymore. "trim" is particularly useful for email and password. Please restore it